backend/stakepoold/stakepool/client.go

```diff
@@ -5,7 +5,9 @@ import (
 	"sync"
 	"time"
 
-	"github.com/decred/dcrd/rpcclient/v5"
+	walletRPC "decred.org/dcrwallet/rpc/client/dcrwallet"
```

Comment:
I have never seen a package name use capital letters so this seems a bit not idomatic.

Unless there is a technical limitation to overcome, I'd recommend using the real package name here (`dcrwallet`).